docs/4.2/installation.md

```diff
@@ -113,16 +113,16 @@ $ which teleport
 
 ### Windows (tsh client only)
 
-As of version v4.1.0 we have `tsh` client binary available for Windows 64-bit
+As of version v4.2.1 we have `tsh` client binary available for Windows 64-bit
```

Comment:
We’ve had the Windows client available since v3.1 or earlier I think - this isn’t one that should be updated.

Reply:
I searched through release history and found support starting from v3.0.1, so I used this version instead.